libkbfs/bare_root_metadata_v3.go

```diff
@@ -193,15 +188,6 @@ func (md *BareRootMetadataV3) IsValidRekeyRequest(
 		// Copy mismatch.
 		return false, nil
 	}
-	writerSigInfoEqual, err := kbfscodec.Equal(codec,
```

Comment:
Ah I missed that there was already a `RootMetadataSigned.IsValidAndSigned`, for some reason I thought the server was calling it directly on the RMD.  All makes sense thanks.